samples/outbox/store-and-forward/Version_5/Sender/app.config

```diff
@@ -1,17 +0,0 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
In that case I think it would be the same for both, right? Because v2 works with v5 only and v3 with v6. So I'd go with transports versions.